src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/respondent/RespondentTurnOnNotificationsPersonalisation.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.respondent;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.HearingCentre;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.CustomerServicesProvider;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.HomeOfficeEmailFinder;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.PersonalisationProvider;
+
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static java.util.Objects.requireNonNull;
+
+
+@Service
+public class RespondentTurnOnNotificationsPersonalisation implements EmailNotificationPersonalisation {
+
+    private final String homeOfficeTurnOnNotificationsBeforeListingTemplateId;
+    private final String homeOfficeTurnOnNotificationsAfterListingTemplateId;
+    private final String iaExUiFrontendUrl;
+    private final PersonalisationProvider personalisationProvider;
+    private final CustomerServicesProvider customerServicesProvider;
+    private final HomeOfficeEmailFinder hoEmailAddressFinder;
+
+
+
+    public RespondentTurnOnNotificationsPersonalisation(
+            @NotNull(message = "endAppealHomeOfficeTemplateId cannot be null")
```

Comment:
Can you rename these "endAppeal *" values

Reply:
Change made